client/consensus/aura/src/lib.rs

```diff
@@ -660,6 +660,30 @@ mod tests {
 		TestClient,
 	};
 
+	#[derive(Clone)]
+	struct TestInherentDataProvider;
+
+	const ERROR_TO_STRING: &str = "Found error!";
+	const TEST_INHERENT_0: sp_inherents::InherentIdentifier = *b"testinh0";
```

Comment:
super-nit: I thought this was a typo for `testing` at first. :P Maybe:

```suggestion
	const TEST_INHERENT_0: sp_inherents::InherentIdentifier = *b"test_inh_0";
```

Reply:
I just copied and pasted this from a different test :shrug:

Reply:
Okay nevermind. 👍